src/Commands/Remote/DrushCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Remote;
+
+/**
+ * Command to proxy drush commands on an Environment using SSH
+ *
+ * @package Pantheon\Terminus\Commands\Remote
+ */
+class DrushCommand extends SSHBaseCommand
+{
+    /**
+     * @inheritdoc
+     */
+    protected $command = 'drush';
+
+    /**
+     * @inheritdoc
+     */
+    protected $unavailable_commands = [
+        'sql-connect' => 'connection:info --field=mysql_command',
+        'sql-sync'    => '',
+    ];
+
+    /**
+     * Run arbitrary drush commands on a site environment
+     *
+     * @command remote:drush
+     * @aliases   drush
+     *
+     * @authenticated
+     *
+     * @param string $site_env_id Name of the environment to run the drush command on.
+     * @param array $drush_command Drush command to invoke on the environment
```

Comment:
Anyone else notice that the param descriptions don't show up for any parameter but the first in `--help` output?